concourse/pipelines/deploy-parameters.yml

```diff
@@ -0,0 +1,2 @@
+workspace: bob
+govuk_infrastructure_branch: test_workspace
```

Comment:
Should we change these back to `default` and `main`?

Reply:
yeah, that's the issue with playing around too much with it. thanks for catching this.